packages/checkout/widgets-lib/src/widgets/wallet/components/BalanceItem/BalanceItem.tsx

```diff
@@ -84,7 +84,10 @@ export function BalanceItem({
 
   return (
     <MenuItem testId={`balance-item-${balanceInfo.symbol}`} emphasized>
-      <MenuItem.FramedIcon icon="Coins" circularFrame />
+      <MenuItem.FramedImage
+        imageUrl={balanceInfo.iconLogo}
```

Comment:
@dreamoftrees  can we double check if this is meant to be icon or iconLogo?

Reply:
ah we do the mapping in tokenBalances function

Reply:
The underlying TokenInfo is `icon` but in the `BalanceInfo` class we are mapping it to `iconLogo` .. ideally its consistent using `icon` but didn't want to refactor that right now so left it mapping across.

Reply:
Ok refactored this to use `icon` consistently.. feels a lot cleaner.